build.sbt

```diff
@@ -474,7 +473,6 @@ lazy val rspace = (project in file("rspace"))
       guava
     ),
     /* Tutorial */
-    tutTargetDirectory := (baseDirectory in Compile).value / ".." / "docs" / "rspace",
```

Comment:
Do we have generated files checked-in in the repo?

_Tutorial_ comment should also be removed, right?

Reply:
```
sbt:rchain> rspace/tutSourceDirectory
[info] Total time: 4 ms
[info] /${HOME}/rchain/rchain/rspace/src/main/tut
[info] Total time: 5 ms
```
The source directory is not exist. It should be some old developer's work.

The output file is https://github.com/rchain/rchain/blob/dev/docs/rspace/Tutorial.md.

@tgrospic Do you think we should remove https://github.com/rchain/rchain/blob/dev/docs/rspace/Tutorial.md?

Reply:
Oh, I see now. Tutorial.md is not generated but type checked by tut so it should stay.

I added a task to Core board to include replacement mdoc later.

Reply:
Also Tutorial.md source file (tut) is deleted here https://github.com/rchain/rchain/pull/2629 ???